integration/controllers/carpool.ctrl.spec.ts

```diff
@@ -300,6 +314,42 @@ describe('Carpool controller', () => {
       .finally(done);
   });
 
+  it('should update a carpool', (done) => {
+    var carpool:models.Carpool;
+    var carpoolUpdate = {
+      'name': 'name updated',
+      'description': 'description updated'
+    };
+
+    function test(status) {
+      expect(status).toBe(204);
+
+      var updatedCarpool = CarpoolSvc.getCarpoolByID(carpool.id);
+      expect(updatedCarpool['name']).toBe(carpoolUpdate.name);
+      expect(updatedCarpool['description']).toBe(carpoolUpdate.description);
+    }
+
+    var req = <restify.Request> {params: {}};
+    var res = <restify.Response> {send: test};
+    req.body = carpoolUpdate;
+
+    CarpoolSvc.createCarpool('update this name', 'PSU', 'update this description', owner.userName)
+      .then((_carpool) => {
+        carpool = _carpool;
+        req.params.carpoolid = _carpool.id;
+      })
+      .then(() => {
+        return CarpoolSvc.updateCarpool(carpool.id, carpoolUpdate);
```

Comment:
It doesn't seem to. We ended up calling both in the ``getCarpools()`` test as well.